frontend/src/__tests__/cypress/cypress/tests/e2e/distributedWorkloadMetrics/testWorkloadMetricsDefaultPageContents.cy.ts

```diff
@@ -0,0 +1,156 @@
+import yaml from 'js-yaml';
+import { HTPASSWD_CLUSTER_ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import {
+  createOpenShiftProject,
+  deleteOpenShiftProject,
+} from '~/__tests__/cypress/cypress/utils/oc_commands/project';
+import {
+  globalDistributedWorkloads,
+  projectMetricsTab,
+  distributedWorkloadStatusTab,
+} from '~/__tests__/cypress/cypress/pages/distributedWorkloads';
+import {
+  createKueueResources,
+  deleteKueueResources,
+} from '~/__tests__/cypress/cypress/utils/oc_commands/distributedWorkloads';
+import type { WorkloadMetricsTestData } from '~/__tests__/cypress/cypress/types';
+
+describe('Verify Workload Metrics Default page Contents', () => {
+  let testData: WorkloadMetricsTestData;
+
+  before(() => {
```

Comment:
You may need to update it to use `retryableBefore` after rebasing.

Reply:
@manosnoam Thanks for review updated to use `retryableBefore` also updated tags name to `@Sanity`